### ISSUE TITLE
Error when updating screen mid-navigation

### ISSUE THREAD

**Issue by SatyaFariz** [grahammendick/navigation#274]
I always got this error when i'm trying to update my screen mid-navigation.

![54f34f5e-5313-4e71-9662-015c6c96e438](https://user-images.githubusercontent.com/25284201/62390277-13939a00-b58c-11e9-9715-9047ab182bc9.jpg)

This is the code

```javascript
const Component = props => (
  <QueryRenderer
    variables={{ id: props.id }}
    query={query}
    environment={environment}
    render={({ error, props, retry }) => {
      if(error) {
        return (
          <ErrorView/>
        )
      } else if(props) {
          // got an error here
          return (
             <ComponentWithNoErrors/>
          )
      }

      return <LoadingView/>
    }}
  />
)
```

I have also made sure that ```ComponentWithNoErrors ``` has no errors as a standalone component.

**Comment by grahammendick** [comment]
Are you changing the first child of a screen? The Navigation route expects the first child of each screen to remain unchanged. Wrap each screen in a ScrollView or SafeAreaView and only change the components underneath this container.

**Comment by SatyaFariz** [comment]
Ah you're right. I wrap the component in a ScrollView and the errors gone. But why ScrollView and why View won't work? I'm a little worried that if I have too many nested ScrollView it will affect the perf of the app.

**Comment by grahammendick** [comment]
Better to use a `SafeAreaView` than a `View` so that the content doesn't overlap the boundaries on iOS. If you're just using Android then you can safely use `View`

**Comment by SatyaFariz** [comment]
I'm using Android. Using ordinary View doesn't, but using ScrollView works. It's weird.

**Comment by grahammendick** [comment]
Wrapping the screen in a `View` will fix the error, too. Just make sure you don't change the View because the Navigation router expects the first child to remain unchanged

**Comment by SatyaFariz** [comment]
Turns out I need to set ```collapsable``` to false if I want to use ```View```. I'm closing this.

**Comment by grahammendick** [comment]
Thanks for letting me know. I forgot that layout-only `Views` can be removed from the hierarchy

**Comment by grahammendick** [comment]
I've just released navigation-react-native v5.5.0. You don't need to wrap your screens in a `ScrollView` or `View` anymore. Thanks for raising the issue
